src/track/beatgrid.cpp

```diff
@@ -206,10 +244,7 @@ double BeatGrid::findNthBeat(double dSamples, int n) const {
         n = n + 1;
     }
 
-    double dResult = floor(dClosestBeat + n * m_dBeatLength);
-    if (!even(static_cast<int>(dResult))) {
-        dResult--;
-    }
+    double dResult = dClosestBeat + n * m_dBeatLength;
```

Comment:
This is a good step towards a double precision transport engine. 

However we need to double check that all using code is aware of that. 
We can easily introduce bugs here if the using code for example uses a compare. 

This may also effect user data, because a quantize cue point will not longer be on the beat position 

Maybe it is a good idea to separate this change into a separate Commit or PR. 
What do you think?